app/src/main/java/org/simple/clinic/scheduleappointment/facilityselection/FacilitySelectionActivity.kt

```diff
@@ -65,7 +69,9 @@ class FacilitySelectionActivity : AppCompatActivity(), FacilitySelectionUi, Faci
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_select_facility)
+
+    binding = ActivitySelectFacilityBinding.inflate(layoutInflater)
```

Comment:
Is there a reason why we are not clearing the binding in some cases?

Reply:
Same comment as above. We don't need to do that for Activities. We only need to clear it for Fragments and Views, since those are placed inside an Activity. But `Activity` will automatically clear all references as soon as an activity is cleared from the memory (AKA when activity is closed).